### PR TITLE
[IT-898] Remove IAM users

### PR DESCRIPTION
**Pull request by zaro0508**
This makes it so that the only way to access AWS bridge dev and bridge prod accounts will be thru Jumpcloud.

* Setup iOS developer access thru jumpcloud
* Remove all IAM Users

### CODE REVIEW

File: config/prod/jumpcloud-idp.yaml [hunk 1]
```diff
@@ -6,5 +6,7 @@ stack_tags:
   OwnerEmail: "it@sagebase.org"
 parameters:
   BridgeprodAdminMetadata: !file_contents idp/jumpcloud/bridgeprod-admin.xml
+  BridgeprodiOSDeveloperMetadata: !file_contents idp/jumpcloud/bridgeprod-ios-developer.xml
 dependencies:
   - "prod/cfn-cr-saml-provider.yaml"
+  - "develop/iam-policies.yaml"
```
Comment (ahayden) [inline config/prod/jumpcloud-idp.yaml:12]: This path include should be `prod/iam-policies.yaml`

File: templates/iam-policies.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Description: IAM Policies for AWS
+AWSTemplateFormatVersion: 2010-09-09
+Resources:
+  iOSBucketManagedPolicy:
```
Comment (ahayden) [inline templates/iam-policies.yaml:4]: Is this policy used anywhere? If not, please remove.

File: templates/bridge.yaml [hunk 3]
```diff
@@ -40,115 +37,6 @@ Resources:
...
-    Properties:
-      UserName: yongjoon.oh@sagebase.org
-      Groups:
-        - !Ref AWSIAMAllUsersGroup
-        - !Ref AWSIAMDeveloperUsersGroup
-      LoginProfile:
-        Password: !Ref InitNewUserPassword
-        PasswordResetRequired: true
-       
-  AWSIAMDeveloperUsersGroup:
```
Comment (zaro0508) [inline templates/bridge.yaml:144]: We need an IDP for this group of users

File: templates/iam-policies.yaml (hunk 2 above)
Comment (zaro0508) [reply templates/iam-policies.yaml:4]: yes, it should be used in the jumpcloud-idp.yaml files.

File: config/prod/jumpcloud-idp.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/jumpcloud-idp.yaml]: fixed

File: templates/iam-policies.yaml (hunk 2 above)
Comment (zaro0508) [reply templates/iam-policies.yaml:4]: fixed

File: templates/prod/jumpcloud-idp.yaml [hunk 4]
```diff
@@ -30,6 +33,31 @@ Resources:
...
+      Name: "bridgeprod-ios-developer"
+      Metadata: !Ref BridgeprodiOSDeveloperMetadata
+      URL: ""
+  BridgedprodiOSDeveloperSamlProviderRole:
+    Type: AWS::IAM::Role
+    Properties:
+      MaxSessionDuration: 28800
+      RoleName: !GetAtt BridgedprodiOSDeveloperSamlProvider.Name
+      ManagedPolicyArns:
+        - 'Fn::ImportValue': !Sub '${AWS::Region}-iam-policies-CfnDeployerServiceManagedPolicyArn'
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply templates/prod/jumpcloud-idp.yaml]: fixed

File: templates/bridge.yaml (hunk 3 above)
Comment (zaro0508) [reply templates/bridge.yaml:144]: added bridgedev-developer and bridgeprod-developer

File: templates/develop/jumpcloud-idp.yaml [hunk 5]
```diff
@@ -30,6 +33,31 @@ Resources:
...
+      Name: "bridgedev-ios-developer"
+      Metadata: !Ref BridgedeviOSDeveloperMetadata
+      URL: ""
+  BridgedeviOSDeveloperSamlProviderRole:
+    Type: AWS::IAM::Role
+    Properties:
+      MaxSessionDuration: 28800
+      RoleName: !GetAtt BridgedeviOSDeveloperSamlProvider.Name
+      ManagedPolicyArns:
+        - 'Fn::ImportValue': !Sub '${AWS::Region}-iam-policies-CfnDeployerServiceManagedPolicyArn'
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply templates/develop/jumpcloud-idp.yaml]: fixed

**Pull request merged**